include/drivers/spi.h

```diff
@@ -703,6 +703,22 @@ static inline int z_impl_spi_release(const struct device *dev,
 	return api->release(dev, config);
 }
 
+/**
+ * @brief Release the SPI device specified in @p spi_dt_spec.
+ *
+ * This is equivalent to:
+ *
+ *     spi_release(spec->bus, &spec->config);
+ *
+ * @param spec SPI specification from devicetree
+ *
+ * @retval a value from spi_release()
```

Comment:
Usage of `@retval` is incorrect here, see https://www.doxygen.nl/manual/commands.html (`@retval VALUE DESCRIPTION`). `@return` may be a better fit here.